roles/contiv_network/tasks/fwd_tasks.yml

```diff
@@ -0,0 +1,3 @@
+- name: set fwd mode
```

Comment:
yeah, as of now we have been keeping the common tasks in `main.yml` while deployment specific tasks (like the ones for aci) in respective files. Since this seems like common task I was thinking of leaving it in `main.yml`. Are  you okay with making this change?